### PR TITLE
Added Additional attributes to Condition Resource

### PR DESCRIPTION
**Pull request by RohitKandimalla**
Pull requests into Bonnie require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

Jira ticket: [MAT-2847](https://jira.cms.gov/browse/MAT-2847)

[Condition Resource](https://hl7.org/fhir/Condition.html)

Verified that when an attribute of type 'reference' is added, associated child cards are also populated.
image

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] This PR is into the correct branch.
- [x] JIRA ticket for this PR:
- [ ] JIRA ticket links to this PR
- [x] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
- [ ] If UI changes have been made, google WAVE plug-in has been executed to ensure no 508 issues were introduced.
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass (remember to update Gemfile when applicable)
- [ ] Test fixtures updated and documented as necessary ( see [internal wiki](https://gitlab.mitre.org/bonnie/internal-documentation/wikis/testing#test-fixtures) )
- [x] Code coverage has not gone down and all code touched or added is covered. 
     * In rare situations, this may not be possible or applicable to a PR. In those situations:
         1. Note why this could not be done or is not applicable here: 
         2. Add TODOs in the code noting that it requires a test
         3. Add a JIRA task to add the test and link it here: 
- [ ] Automated regression test(s) pass

If JIRA tests were used to supplement or replace automated tests:
- [ ] JIRA test links:
- [ ] Justification for using JIRA tests:
- [ ] JIRA tests have been added to sprint

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [ ] JIRA tests have been run and pass
- [ ] You agree with the justification for use of JIRA tests or have provided input on why you disagree

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [ ] JIRA tests have been run and pass
- [ ] You agree with the justification for use of JIRA tests or have provided input on why you disagree

### CODE REVIEW

File: app/assets/javascripts/helpers/fhir_value_sets.js.coffee [hunk 1]
```diff
@@ -100,6 +100,43 @@
     "id": "2.16.840.1.113883.4.642.3.166"
   }
 
+  @CONDITION_SEVERITY_VS = {
+	  "resourceType": "ValueSet",
```
Comment (ghost) [inline app/assets/javascripts/helpers/fhir_value_sets.js.coffee]: Looks like those are TABs. We should be using spaces for indentation. Could you adjust your IntelliJ settings?

Comment (RohitKandimalla) [reply app/assets/javascripts/helpers/fhir_value_sets.js.coffee]: Fixed

**Review by ghost — DISMISSED** [review]

File: app/assets/javascripts/helpers/fhir_value_sets.js.coffee [hunk 2]
```diff
@@ -100,6 +100,43 @@
     "id": "2.16.840.1.113883.4.642.3.166"
   }
 
+  @CONDITION_SEVERITY_VS = {
+    "resourceType": "ValueSet",
```
Comment (ghost) [inline app/assets/javascripts/helpers/fhir_value_sets.js.coffee]: I would suggest moving it to a separate file under value_sets..

**Review by ghost — DISMISSED** [review]

File: app/assets/javascripts/helpers/fhir_value_sets.js.coffee (hunk 2 above)
Comment (RohitKandimalla) [reply app/assets/javascripts/helpers/fhir_value_sets.js.coffee]: Created a separate file for the Value Set

**Review by ghost — APPROVED** [review]

**Pull request merged**